apps/frontend/src/app/3_organisms/RewardHistory/components/RewardsEarnedHistory/RewardsEarnedHistory.constants.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { t } from 'i18next';
+
+import { TransactionIdRenderer } from '../../../../2_molecules/TransactionIdRenderer/TransactionIdRenderer';
+import { translations } from '../../../../../locales/i18n';
+import { RewardsEarnedHistoryItem } from '../../../../../utils/graphql/rsk/generated';
+import { dateFormat } from '../../../../../utils/helpers';
+import { getTransactionType } from './RewardsEarnedHistory.utils';
+import { RewardsAmountRenderer } from './components/RewardsAmountRenderer';
+
+export const COLUMNS_CONFIG = [
+  {
+    id: 'timestamp',
+    title: t(translations.common.tables.columnTitles.timestamp),
+    cellRenderer: (tx: RewardsEarnedHistoryItem) => dateFormat(tx.timestamp),
+    sortable: true,
+  },
+  {
+    id: 'stabilityPoolOperation',
+    title: t(translations.common.tables.columnTitles.transactionType),
+    cellRenderer: (tx: RewardsEarnedHistoryItem) =>
+      getTransactionType(tx.action),
+  },
+  {
+    id: 'amount',
+    title: t(translations.common.tables.columnTitles.amount),
+    cellRenderer: (tx: RewardsEarnedHistoryItem) => (
+      <RewardsAmountRenderer {...tx} />
+    ),
+  },
+  {
+    id: 'transactionID',
+    title: t(translations.common.tables.columnTitles.transactionID),
+    cellRenderer: (tx: RewardsEarnedHistoryItem) => (
+      <TransactionIdRenderer
+        hash={tx.id.split('/')[0]}
```

Comment:
This is not working correctly in `Staking revenue rewards` (not sure about `Staking subsidies rewards` because I don't have any).

<img width="2060" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/5535ffbd-7274-48f7-8c69-48d0cc97891d">